src/components/select/select.vue

```diff
@@ -0,0 +1,240 @@
+    e.key != 'Tab'
+  ) {
+    if(e.key == 'Space') {
+      searchRef.value = searchRef.value + ' ';
+    } else {
+      searchRef.value = searchRef.value + e.key;
+    }
+  
+    const index = optionsList.value.findIndex(option => option.text.toLowerCase().includes(searchRef.value.toLowerCase()))
+    if(index !== -1) {
+      optionRefs.value[index].focus()
+    }
+
+    if (searchTimer) {
+      return;
+    }
+
+    searchTimer = setTimeout(() => {
+      searchRef.value = '';
+      searchTimer = null;
+    }, 2500);
+  }
+};
+
+const handleSelectOption: (option: string, value: string) => void = (option, value) => {
+  selectedOption.value = option;
+  showOptions.value = false;
+  document.getElementById(selectContainerId.value)?.focus();  
+
+  selectRef.value.removeAttribute('data-show');
+  emit('change', value);
+};
+
+const handleEnterSelect: (option: string, value: string, e: KeyboardEvent) => void = ( option, value, e) => {
+  e.stopPropagation();
+  selectRef.value.removeAttribute('data-show');
+  handleSelectOption(option, value);
+};
+
+// Close option list on click outside the component
+useDetectOutsideClick(componentRef, () => {
+  showOptions.value = false;
+  selectRef.value.removeAttribute('data-show');
+});
+</script>
+
+<template>
+  <section
+    :id="selectId"
+    ref="componentRef"
+    :tabindex="-1"
+    class="select"
+    :focusable="false"
+  >
+    <div
+      :id="selectContainerId"
+      class="select-container"
+      :tabindex="0"
+      @click="handleShowOptions"
+      @keydown.enter="handleEnterShow"
+      @keydown="handleSearch($event)"
+    >
+      <p v-if="selectedOption.length > 0" class="active--option subtitle2">
+        {{ selectedOption }}
+      </p>
+      <label
+        class="select--label subtitle2"
+        :class="[`label-${labelPosition}`]"
+        for="select-input"
+        :data-active="[showOptions && labelPosition == 'inside' ? true : false]"
+        :tabindex="-1"
+      >
+        {{ props.label }}
+      </label>
+      <div class="select--button">
+        <Icon
+          :class="[showOptions ? 'options-active' : null]"
+          icon="caret-down"
+          :size="16"
+        />
+      </div>
+    </div>
+    <div
+      id="option__container"
+      ref="selectRef"
+      :class="selectId"
+      @keydown="handleSearch($event)"
+    >
+      <div
+        v-for="(option, index) in optionsList"
+        :id="`option-${option.id}`"
+        :ref="(el) => { optionRefs[index] = el as HTMLElement }"
+        :key="index"
+        :class="
+          selectedOption == option.text ? 'option--selected option' : 'option'
+        "
+        tabindex="0"
+        @click="handleSelectOption(option.text, option.value)"
+        @keydown.enter="handleEnterSelect(option.text, option.value, $event)"
+        @keyup.escape="handleShowOptions"
+        @keydown.tab="useTabTrap($event)"
+        @keyup="useRemoveRecordedStroke($event)"
+      >
+        {{ option.text }}
+      </div>
```

Comment:
Change the font-size to something similiar you have on the dropdown,

Reply:
Added font-size class.